packages/extension-chakra-storefront/config/default.json

```diff
@@ -186,5 +186,10 @@
         "locale": "path",
         "showDefaults": true,
         "interpretPlusSignAsSpace": false
+    },
+    "resourceTypeToComponentMap": {
+        "category": "ProductList",
+        "product": "ProductDetail",
+        "content_asset": "ProductList"
```

Comment:
I'd expect that the resourceType here is capitalized like the return value before in the [API docs](https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-seo?meta=getUrlMapping)

Reply:
Interestingly the actual API returns the resource type in lower case
<img width="1062" alt="Screenshot 2025-05-01 at 10 58 02 AM" src="https://github.com/user-attachments/assets/85d7f6a8-6e16-41b0-9951-8f5937c9ade8" />